### PR TITLE
chore: bump version to 20.0

### PR DESCRIPTION
**Pull request by platosha**

### CODE REVIEW

**Comment by Artur-** [comment]
Why .1 suddenly?

**Comment by haijian-vaadin** [comment]
> Why .1 suddenly?

We have a new feature coming up https://github.com/vaadin/spring/pull/931. We are in a situation where the PR looks fine, but we need a new branch.

**Comment by Artur-** [comment]
> > Why .1 suddenly?
> 
> We have a new feature coming up #931. We are in a situation where the PR looks fine, but we need a new branch.

Sure, but why not 20.0

**Comment by haijian-vaadin** [comment]
> > > Why .1 suddenly?
> > 
> > 
> > We have a new feature coming up #931. We are in a situation where the PR looks fine, but we need a new branch.
> 
> Sure, but why not 20.0

ok, just checked, so it has always been bumping the major versions in the past, disregard the changes. So @platosha, let's use 20.0?

**Comment by platosha** [comment]
Ok, bumped to 20.0.

**Review by haijian-vaadin — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -26,7 +26,8 @@ See other branches for other framework versions:
 * 16.0 for Vaadin 19
 * 17.0 for Vaadin 20 
 * 18.0 for Vaadin 21 
-* master (currently 19.0) for upcoming major Vaadin versions
+* 19.0 for Vaadin 22
+* master (currently 19.1) for upcoming major Vaadin versions
```
Comment (platosha) [inline README.md:30]:
```suggestion
* master (currently 20.0) for upcoming major Vaadin versions
```
forgot to change this number also in the README

**Review by haijian-vaadin — APPROVED** [review]

**Pull request merged**